test/unit/controllers/InventoryLinkingExportControllerSpec.scala

```diff
@@ -191,12 +191,12 @@ class InventoryLinkingExportControllerSpec extends UnitSpec
     }
 
 
-    "respond with status 400 and conversationId in header for a processed valid non-CSP request with eori id that doesn't match our records" in new SetUp() {
+    "respond with status 202 and conversationId in header for a processed valid non-CSP request with eori id that doesn't match our records" in new SetUp() {
```

Comment:
please remove `with eori id that doesn't match our records`, it is misleading now